apps/astarte_realm_management/lib/astarte_realm_management/c_system.ex

```diff
@@ -42,7 +42,7 @@ defmodule CSystem do
   end
 
   @spec wait_schema_agreement(integer()) ::
-          {:ok, Astarte.RealmManagement.UUID.t()} | {:error, :timeout}
+          {:ok, Astarte.DataAccess.UUID.t()} | {:error, :timeout}
```

Comment:
The `Astarte.DataAccess.UUID.t()` type represents a "raw" Ecto UUID as they are used in Astarte e.g. for device IDs (see more [here](https://github.com/astarte-platform/astarte_data_access/blob/4fa71dafabdcb9b36ca40ea4831e2c0caf610b11/lib/astarte_data_access/uuid.ex#L20-L23)). However, here (and in the other 3 queries below) you're retrieving Ecto UUIDs, so I guess the correct type would be `Ecto.UUID.t()`